### PR TITLE
Remove deprecated warning on DebugClassLoader

### PR DESCRIPTION
**Pull request by frenchcomp**
Hi,
This PR is about to support new DebugClass component of Symfony available in symfony/error-handler when it is available. (The legacy DebugClass is deprecated since 4.4).

Thanks

### CODE REVIEW

File: tests/FunctionalTest/ContainerInteractionTest.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ class ContainerInteractionTest extends AbstractFunctionalTest
     /**
      * @test Get a Symfony entry from PHP-DI's container
      */
-    public function phpdi_should_get_entries_from_symfony()
+    public function phpdiShouldGetEntriesFromSymfony()
```
Comment (mnapoli) [inline tests/FunctionalTest/ContainerInteractionTest.php]: Could you restore the original name? (including in the other tests)

File: tests/FunctionalTest/ContainerAwareTest.php [hunk 2]
```diff
@@ -27,6 +27,6 @@ public function testContainerAware()
         /** @var ContainerAwareController $class */
         $class = $container->get(ContainerAwareController::class);
 
-        $this->assertSame($container, $class->container);
+        $this::assertSame($container, $class->container);
```
Comment (mnapoli) [inline tests/FunctionalTest/ContainerAwareTest.php]: These code-style changes are unrelated to the PR, could you revert them?

File: tests/UnitTest/SymfonyContainerBridgeTest.php [hunk 3]
```diff
@@ -11,39 +11,41 @@
...
-class SymfonyContainerBridgeTest extends \PHPUnit_Framework_TestCase
+class SymfonyContainerBridgeTest extends TestCase
 {
     public function testHasFallback()
     {
         $wrapper = new SymfonyContainerBridge();
 
         $fallback = $this->getMockForAbstractClass(ContainerInterface::class);
-        $fallback->expects($this->once())
+        $fallback->expects(self::once())
```
Comment (mnapoli) [inline tests/UnitTest/SymfonyContainerBridgeTest.php]: Same here.

**Review by mnapoli — COMMENTED** [review]
Thanks for the update!

I have added a few comments, once those are fixed let's merge 👍

File: tests/FunctionalTest/ContainerAwareTest.php (hunk 2 above)
Comment (frenchcomp) [reply tests/FunctionalTest/ContainerAwareTest.php]: Assert method are static.

File: tests/UnitTest/SymfonyContainerBridgeTest.php (hunk 3 above)
Comment (frenchcomp) [reply tests/UnitTest/SymfonyContainerBridgeTest.php]: Idem Once method are static.

File: tests/FunctionalTest/ContainerInteractionTest.php (hunk 1 above)
Comment (frenchcomp) [reply tests/FunctionalTest/ContainerInteractionTest.php]: Done

**Comment by frenchcomp** [comment]
Sorry, my mind bugged and i forgot to create branches by PR. Il will create them tomorow evening.

**Comment by frenchcomp** [comment]
I restored phpunit 4 and original test, I will push a new PR after the merge of this one :)

**Comment by mnapoli** [comment]
I'm sorry there are too many changes in this PR, without explanations I have to assume they are unrelated. It's not possible to move forward at this time, I'll be closing this for now.

**Comment by frenchcomp** [comment]
I understand, no worries. Since then I have completely rework the behavior to run as bundle without having to overload the Symfony kernel.
Have a nice day :)

**Comment by mnapoli** [comment]
Thanks, you too!
